lib/Album/AlbumInfo.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2022 Robin Appelman <robin@icewind.nl>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Photos\Album;
+
+class AlbumInfo {
+	private int $id;
+	private string $userId;
+	private string $title;
+	private string $location;
+	private int $created;
```

Comment:
Not needed anymore. Album's date is now dynamic based on its content: `oldest item's month and year - newest item's month and year`
https://github.com/nextcloud/photos/pull/1142/files#diff-9706d467ff74e216f41e0da8dd6424754754a63018ff15b3e7fadcef9d33f811R118-R134